Formula/g/gpgme.rb

```diff
@@ -37,20 +38,43 @@ def install
     # hardcoded, the Arch Linux patch that changed 3.9 to 3.10 can't detect 3.11
     inreplace "configure", /# Reset everything.*\n\s*unset PYTHON$/, ""
 
-    # Uses generic lambdas.
-    # error: 'auto' not allowed in lambda parameter
-    ENV.append "CXXFLAGS", "-std=c++14"
-
     # Use pip over executing setup.py, which installs a deprecated egg distribution
     # https://dev.gnupg.org/T6784
     inreplace "lang/python/Makefile.in",
               /^\s*\$\$PYTHON setup\.py\s*\\/,
               "$$PYTHON -m pip install --use-pep517 #{std_pip_args.join(" ")} . && : \\"
 
+    # Required for Qt 6 bindings
+    ENV.append "CXXFLAGS", "-std=c++17"
+
+    ENV["PKG_CONFIG_PATH"] = [
+      "#{Formula["qt@6"].libexec}/lib/pkgconfig",
+      "#{HOMEBREW_PREFIX}/lib/pkgconfig",
+    ].join(":")
+
     system "./configure", "--disable-silent-rules",
                           "--enable-static",
+                          "--enable-languages=cl,cpp,python,qt6",
```

Comment:
Not at the moment. But it will open the possibility to add Qt-based software like Kleopatra and KMail from KDE.
I do run my own build of Kleopatra on a daily basis. And this patch is actually a port from my personal tap.
This would be the first step to bring Kleopatra and other software into `homebrew-core` someday.

Reply:
Sonoma bottle works just fine. And new bottles build against Qt are still build. E.g., gnuplot

Reply:
Qt story is dropped for now. I will recap it in an additional comment